### PR TITLE
Edited Vibhor Verma's Profile 

### PR DESCRIPTION
**Pull request by mrthunjai**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a Preview link and screenshot showning after and before the changes.
- [x] Images are `240 x 240` [w x h].
- [x] Included a description of change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

**Review by ParthS007 — CHANGES_REQUESTED** [review]
@mjd220903  PLease provide a preview link 👍

**Comment by mrthunjai** [comment]
how do i provide it ??

**Comment by mrthunjai** [comment]
please tell me so that i can provide it @ParthS007

**Comment by mrthunjai** [comment]
https://github.com/fossasia/gci17.fossasia.org/pull/691/files/eba12fbb03e143e60c7e7ad71836235fba1d3e40

**Comment by mrthunjai** [comment]
screen shot 2018-01-01 at 12 50 56 am

This is before

screen shot 2018-01-01 at 12 51 38 am

This is after

**Comment by mrthunjai** [comment]
Is it fine @ParthS007 
If it is not let me know

**Comment by ParthS007** [comment]
@mjd220903  You are doing everything correct but your PR contains previous commits (like this PR also have changes which add your profile also but ) so please remove please update your local and fork and then make a new PR 👍

**Comment by mrthunjai** [comment]
can you guide me i don't understand

**Comment by mrthunjai** [comment]
**@ParthS007  are you there ??**

**Comment by mjd220903** [comment]
@ParthS007

**Comment by harshit98** [comment]
@mjd220903 Don't open and close PR again and again.

- Change the name of Vibhor Verma's image to `vibhor.jpg`.

**Comment by mrthunjai** [comment]
@harshit98 should i change to code or the photo name ??

File: _data/mentors.yml [hunk 1]
```diff
@@ -215,7 +215,8 @@
 
 - name: Vibhor Verma
   github: vibhcool
-  image: vibhor.jpg
+  twitter: vibhcool
+  image: CsZwA5Xn.jpg
```
Comment (harshit98) [inline _data/mentors.yml]: Change the image name: @mjd220903. Keep it `vibhor.jpg`

**Review by harshit98 — APPROVED** [review]
LGTM! 💯 

@mjd220903 Good work! I'll merge this pull request as soon as it will pass Travis CI.

**Comment by mrthunjai** [comment]
Ok thank you  @harshit98

**Comment by mrthunjai** [comment]
@harshit98 Merge it please as all the checks have passed

**Pull request merged**